product/pom.xml

```diff
@@ -168,18 +174,42 @@
             <type>zip</type>
         </dependency>
         <!--Sample bundles-->
-        <dependency>
+        <!--dependency>
```

Comment:
Rather than commenting-out, if no longer used then let's remove this dependency.